dftimewolf/lib/exporters/timesketch.py

```diff
@@ -19,37 +19,43 @@ class TimesketchExporter(BaseExporter):
     sketch_id: Timesketch Sketch ID the timeline will be added to
   """
 
-  def __init__(self, timesketch_api, incident_id, sketch_id, verbose, paths):
+  def __init__(self, auth, incident_id, sketch_id, verbose, paths):
```

Comment:
-auth +authentication_information or something. But it's better to just pass these args separately, rather than as as tuple.